tests/integration/test_evalml_model_artifact.py

```diff
@@ -0,0 +1,76 @@
+import pandas
+import pickle
+import pytest
+
+import evalml
+
+import bentoml
+from bentoml.utils import cloudpickle
+from bentoml.yatai.client import YataiClient
+from tests.bento_service_examples.evalml_classifier import EvalMLClassifier, EvalMLClassifierCloudpickle
+
+
+@pytest.fixture(scope="session")
```

Comment:
I didn't see other integration tests doing this. But refactoring the other integration test model fixtures to use session scope (instead of the default which is function scope) could save on test runtime.